docs/topics/settings.rst

```diff
@@ -604,6 +604,26 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
+-----------------------
+
+Default: ``False``
+
+Whether or not to allow broken responses, where declared ``Content-Length``
+does not match content sent by the server. If ``False``, these responses
+trigger a ``ResponseFailed`` error. If ``True``, these responses are allowed
```

Comment:
Would it be useful to have an option to retry these responses, but keep the last one if retry attempts failed? Based on your experience, can retries fix this data loss problem, or is it usually just a server configuration issue?